docs/assets/js/charts.js

```diff
@@ -350,23 +354,23 @@ function createTable(table)
 			if (data.length == 0)
 				return;
 
-			var header = d3.select(table)
+			d3.select(table)
 				.append("thead")
 				.append("tr")
 				.selectAll("th")
-					.data(data.columns)
-					.enter()
-					.append("th")
-					.text(function(d) { return d; });
+				.data(data.columns)
+				.enter()
+				.append("th")
+				.text(function(d) { return d; });
```

Comment:
I like the old ident for readability in this specific case.... but having automated code formatting/linting is way more important  😉

Reply:
I agree with you that it looked better before... We could also add an eslint-targetted comment around this code if we want to give eslint a hint to treat this as an exception.